#### Task 1.

Создайте Series, содержащий цифры от 10 до 1000:
``` python
data = pd.Series(list(range(10, 1001)))
```
Что выведет на экран следующий код?
``` python
print(data.loc[10] + data.loc[23] - data.loc[245] + data.iloc[122])
```

#### Solution:

In [1]:
import pandas as pd

In [2]:
data = pd.Series(list(range(10, 1001)))

In [3]:
print(data.loc[10] + data.loc[23] - data.loc[245] + data.iloc[122])

-70


##### Conclusion: -70 is the correct answer!!!

#### Task 2.

Какой индекс у последнего элемента data из предыдущего задания?

#### Solution:

In [5]:
data.tail(1)

990    1000
dtype: int64

##### Conslusion: 990 is the correct answer!!!

#### Task 3.

Какой из нижеперечисленных кодов создаст Series с элементами "Кот", "Собака", "Корова", "Лемур" и индексами 3, 7, 12, 26?

- ``` python
pd.Series(["Кот", "Собака", "Корова", "Лемур"], index = [3,7,12,26])
```
- ``` python
pd.Series([3,7,12,26], index = ["Кот", "Собака", "Корова", "Лемур"])
```
- ``` python
pd.Series([3,7,12,26])
```
- ``` python
pd.Series(["Кот", "Собака", "Корова", "Лемур"])
```

#### Solution:

In [6]:
pd.Series(["Кот", "Собака", "Корова", "Лемур"], index = [3,7,12,26])

3        Кот
7     Собака
12    Корова
26     Лемур
dtype: object

##### Conclusion: the code above is the correct answer!!!